runhouse/resources/hardware/cluster.py

```diff
@@ -2061,3 +2061,27 @@ def list(
             "sky_clusters": sky_live_clusters,
         }
         return clusters
+
+    ###############################
+    # Cluster logs
+    ###############################
+
+    # TODO [SB/JL]: uncomment and update once Rohin's change is merged.
+    # def logs(self, tail: Optional[int] = DEFAULT_SURFACED_LOG_LENGTH):
```

Comment:
similar note as above, can remove the logic and just replace with "pass" for now